conteudo/introducao/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Introdução
+description: Como Instalar e deixar o Brigade funcionando
+section: introducao
+aliases:
+  - /index.md
+  - /intro/index.md
+  - /intro/index/
+---
+
+Se você acabou de conhecer o Brigade ou esta procurando aprender rapidamente a utilizá-lo, você esta no lugar certo. Leia esta página para que rapidamente você tenha o Brigade instalado e funcionando.
+
+## Como a documentação é organizada
+
+A documentação do Brigade é dividida em duas partes que abrangem necessidades diferentes:
+
+A página de introdução a presentada aqui é desenhada para pessoas novas ao projeto do Brigade. Isto não abrange os conceitos em detalhes, mas explica de forma geral como fazer a implantação e operação do Brigade, como criar projetos e como escrever scripts que processam eventos.
```

Comment:
I suggest this changes to better understanding and get more natural

```suggestion
A página de introdução apresentada aqui é direcionada para pessoas novas ao projeto do Brigade. Isto não abrange os conceitos em detalhes, mas explica de forma geral como fazer a implantação e operação do Brigade, como criar projetos e como escrever scripts que processam eventos.
```

Reply:
fixed